src/firefox/package-identifiers.js

```diff
@@ -0,0 +1,10 @@
+/* @flow */
+
+export const browserList = [
+  'org.mozilla.fennec',
+  'org.mozilla.fenix',
```

Comment:
Could you run an actual test with `org.mozilla.fenix.nightly` to verify that it works, and if it does work, add a unit test for it? It is not entirely obvious that `org.mozilla.fenix.nightly` ought to use `org.mozilla.fenix` as the name.

Reply:
Yup, just added the test below